data/sql/grid_transformation_custom.sql

```diff
@@ -55,3 +55,38 @@ INSERT INTO "grid_transformation" VALUES(
 INSERT INTO "area" VALUES('PROJ','BWTA2017','Germany - Baden-Wurtemberg','Germany - Baden-Wurtemberg',47.5,49.83,7.49,10.51,0);
 -- Advertize a 0.8 accuracy slightly better than the 0.9 of BETA2007 for sort purposes
 INSERT INTO "grid_transformation" VALUES('PROJ','BWTA2017','DHDN to ETRS89 (BWTA2017)','DHDN to ETRS89 for Baden-Wurtemberg for ALKIS 2017. Using official BWTA2017 grid but this transformation entry has been created temporarily by PROJ. Accuracy indication not to be considered as authoritative','For applications requiring an accuracy of better than 1 metre.','EPSG','9615','NTv2','EPSG','4314','EPSG','4258','PROJ','BWTA2017',0.89,'EPSG','8656','Latitude and longitude difference file','BWTA2017.gsb',NULL,NULL,NULL,NULL,NULL,NULL,'-',0);
+
+-- Iceland
+
+INSERT INTO "grid_transformation" VALUES(
+    'PROJ','EPSG_5323_TO_EPSG_8089','ISN2004 height to vertical datum ISH2004',
+    NULL,NULL,
+    'PROJ','HEIGHT_TO_GEOGRAPHIC3D','GravityRelatedHeight to Geographic3D',
+    'EPSG','5323', -- source CRS (ISN2004 height)
+    'EPSG','8089', -- target CRS (ISH2004)
+    'EPSG','1120', -- area of use: Iceland - onshore and offshore
+    NULL,
+    'EPSG','8666','Geoid (height correction) model file','Icegeoid_ISN2004.gtx',
```

Comment:
there should be 3 entries in grid_alternatives for Icegeoid_ISN2004.gtx, Icegeoid_ISN93.gtx and Icegeoid_ISN2016.gtx. They should be similar to the one of SWEN17_RH2000.gtx (the important thing is to have inverse_direction = 0 for that particular case as you use a 'GravityRelatedHeight to Geographic3D' method)